packages/dcos-integration-test/extra/test_metrics.py

```diff
@@ -264,7 +264,13 @@ def test_metrics_master_adminrouter_nginx_vts_processor(dcos_api_session):
     node = dcos_api_session.masters[0]
     # Make request to a fine-grained metrics annotated upstream of
     # Admin Router (IAM in this case).
-    dcos_api_session.get('/acs/api/v1/auth/jwks', host=node)
+    r = dcos_api_session.get('/acs/api/v1/auth/jwks')
```

Comment:
Why was the host removed here?
Is this going to make the test flaky on multi-master clusters?